blivetgui/communication/constants.py

```diff
@@ -0,0 +1,5 @@
+
+class ServerInitResponse(object):
+    RUNNING = 1
+    UNUSABLE = 2
+    EXCEPTION = 3
```

Comment:
Any reason why this is not an ``Enum``?

Reply:
I already have other constants without Enum -- https://github.com/storaged-project/blivet-gui/blob/master/blivetgui/dialogs/constants.py (and these can't be enums, because I use them for Gtk functions) and I didn't want to have half enums and half non-enums.

Reply:
I believe you can use the enums for Gtk functions too. But I see your point.